### ISSUE TITLE
Check ONRResource accessibility for accordions

### ISSUE THREAD

**Issue by mcharg** [ONRR/onrr.gov-site#1785]
Run Lighthouse check to see if I can figure out whether the accordions are accessible.

**Comment by mcharg** [comment]
@jkwonatx I'm not seeing any errors come up specific to the accordions, so I'm going to say that as long as they're labeled properly, they should be good to use in the wireframe.
